src/adapters/gitlab.js

```diff
@@ -89,6 +91,8 @@ class GitLab extends Adapter {
       $(GL_HEADER).css('z-index', 3)
       $(GL_SIDEBAR).css('z-index', 1)
       $(GL_TITLE).css('margin-left',  sidebarVisible ? '' : 56)
+      $(GL_CONTENT).parent().css('margin-left', sidebarVisible ? 230 : '')
```

Comment:
Please refactor 2 lines to $(GL_CONTENT).parent().add(GL_MAIN_NAV).css(...). Also, add a comment to point out the specific GL version that this code targets